chainio/clients/builder.go

```diff
@@ -18,7 +17,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-type Config struct {
+type BuildAllConfig struct {
```

Comment:
maybe we should validate this config since it's a free flowing config and can have unintended panics if some configs are not filled?

Reply:
I think panicing inside a constructor is fine, at least it'll be caught pretty early.

Reply:
I mean the problem is more on - you initialize all the other things unnecessarily if the panic happens later. so we might want to make sure config is correct before we start initializing anything.

Reply:
discussed offline.

Reply:
Added in [95482b2](https://github.com/Layr-Labs/eigensdk-go/pull/73/commits/95482b2c9f3eb0bb60971a2672c263add9848d53)